style.css

```diff
@@ -26,6 +26,29 @@ body {
   height: 30px;
 }
 
+.menuList {
+  list-style: none;
+  text-align: center;
+  background-color: #191765;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
```

Comment:
- [ ] Great job implementing the functionality of the Menu 👌💯 but The height of the `menuList` should be the same as the height of the entire page, so I suggest adding `height: 100vh;` or `bottom: 0; top: 0;` to fix this issue. 😉 

- [ ] Also, the logo (your name) should not appear when navMenu opens, so please fix it 👌
-  remember According to the requirements, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) 👌

![2](https://user-images.githubusercontent.com/45231276/179661484-88d3440e-1e94-4ccf-b8eb-7212e6734ce3.PNG)


| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/179701441-be3759ed-cdfb-4bbc-8a93-cc3c8d27e3f4.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/179661296-46deaa75-4e82-44e5-9245-dde0b2aa1af9.PNG" width="500" /> |